풀어본 문제 추가 - UBC*.kt 소스코드 자동 생성하기

HaenaraShin opened the pull request:
UBC*.kt 소스코드 자동 생성하기

해나라는 동일한 규칙의 코틀린 소스코드 파일을 일일히 만드는 대신, 파이썬으로 자동 생성을 해보고자
주어진 함수 목록이 있습니다. 이 함수 명들을 가지고 아래와 같은 코틀린 파일을 생성하고자 합니다.

간단하게 파일 입출력을 이용하여 해결!

HaenaraShin committed:
[풀어볼 문제 추가] UBC*.kt 소스코드 자동 생성하기

해나라는 동일한 규칙의 코틀린 소스코드 파일을 일일히 만드는 대신, 파이썬으로 자동 생성을 해보고자
주어진 함수 목록이 있습니다. 이 함수 명들을 가지고 아래와 같은 코틀린 파일을 생성하고자 합니다.

HaenaraShin committed:
[풀이 추가] UBC*.kt 소스코드 자동 생성하기

간단하게 풀어서 풀어본 문제 목록에 추가

kirdi-1206 commented on UbmcallCreator/UbmcallCreator.md:
```diff
@@ -0,0 +1,94 @@
+# 예제 출력
+
+
+- 파일명 :  UBCsetAttribute.Kt
```
UBC*.kt or UBC*.Kt??

HaenaraShin replied:
.kt 가 맞음
수정하도록 할게

kirdi-1206 commented on UbmcallCreator/UbmcallCreator.md:
````diff
@@ -0,0 +1,94 @@
+}
+```
+
+- 파일명 :  UBCsetAttribute.Kt
````
57줄이랑 같음

HaenaraShin replied:
여기도 수정할게

kirdi-1206 reviewed (commented):
UbmcallCreator.md파일의 대소문자 구분 외에 소스 고칠 곳은 안보입니다.

HaenaraShin committed:
[풀이 수정] UBC*.kt Ubmcall_*.kr 로 수정

Kotlin 네이밍컨벤션에는 맞지 않지만 클래스명이 도저히 구분도 안가고 눈에 들어오질 않아서
UBC*.kt 대신 Ubmcall_*.kt 로 수정
파일 내부 소스코드도 일부 비동기처리로 수정
확장자 .Kt로 대문자로 씌여있는 것도 .kt 로 정정

HaenaraShin commented:
[추가수정](https://github.com/Heavybros/PythonStudy/pull/22/commits/8ac5476553590be7720bcefd0c16e3e0b09a6e15)했습니다. 
리뷰 부탁드립니다.

kirdi-1206 commented:
굿굿 좋아!

kirdi-1206 reviewed (commented):
굿굿

flagman1211 commented:
굿

kirdi-1206 reviewed (commented):
훌륭합니다 ~

kirdi-1206 reviewed (approved):
오킹
